services/core/resources/index.ts

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+import "source-map-support/register";
+import * as cdk from "aws-cdk-lib";
+import { CoreStack } from "./stacks/core-stack";
+
+const app = new cdk.App();
+new CoreStack(app, "crgpt-core-stack", {
+  stackName: "crgpt-core-stack", //todo buildResourceName
+  tags: {
+    environment: "dev",
+    team: "CRGPT Team",
+    compliance: "normal",
+    alarmSeverity: "low",
+    CostCenter: "TBD",
+  },
```

Comment:
Hmm can you remove these props. We wont need them for this app. 

can do something way more chill:
```
new CoreStack(app, `${stage}-crgpt-core-stack`, {
  stage,
  env: { region },
});
```

for now comment out stage and env lines until we have the helpers to populate them.